inc/common/purge.php

```diff
@@ -436,7 +436,21 @@ function do_admin_post_rocket_purge_opcache() {
 		return;
 	}
 
-	rocket_reset_opcache();
+	$reset_opcache = rocket_reset_opcache();
+
+	if ( ! $reset_opcache ) {
+		$op_purge_result = array(
+			'result'  => 'error',
+			'message' => __( 'OPcache purge failed.', 'rocket' ),
+		);
+	} else {
+		$op_purge_result = array(
+			'result'  => 'success',
+			'message' => __( 'OPcache successfully purged', 'rocket' ),
+		);
```

Comment:
We usually use the shorthand syntax for arrays, `[]`instead of `array()`

Reply:
Replaced all array() with the shorthand [] syntax in the files:
`/inc/common/purge.php`
`/inc/admin/ui/notices.php`